apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -171,6 +171,25 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  isInEditMode: function() {
```

Comment:
Yep, you can use similar technique like we use here https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/test/marionette/conversation_test.js#L62